platform/mk2/hal/gps_skytraq/gps_device_skytraq.c

```diff
@@ -29,9 +29,10 @@
 #define MESSAGE_TYPE_BINARY		2
 #define GNSS_NAVIGATION_MODE		GNSS_NAVIGATION_MODE_AUTOMOBILE
 
-#define BAUD_RATE_COUNT 2
+#define BAUD_RATE_COUNT 3
 #define BAUD_RATES 	{			\
 		{921600, 8},			\
+		{115200, 5},            	\
```

Comment:
Ok... here is what I see on RCT:
```
GPS: provisioning attempt
GPS: probing baud rate: 921600
[at] Command timed out
[esp8266] Version info:
        AT version:0.50.0.0(Sep 18 2015 20:55:38)
        SDK version:1.4.0
        compile time:Sep 18 2015 21:30:56
[esp8266] Initialized
[ESP8266 Driver] Device state: 2
[ESP8266 Driver] Checking WiFi Device...
[ESP8266 Driver] Checking WiFi Device...
[ESP8266 Driver] Checking Client
[ESP8266 Driver] Retrieving Wifi Client Info
[ESP8266 Driver] Checking Client
[ESP8266 Driver] Client is inactive.
[ESP8266 Driver] Checking WiFi AP
[ESP8266 Driver] Retrieving Wifi AP Info
[ESP8266 Driver] AP info:
         SSID      : RaceCapture FFF41BEF00F
         Channel   : 11
         Encryption: 0
[ESP8266 Driver] Checking WiFi AP
[ESP8266 Driver] Retrieving Wifi IP info
[ESP8266 Driver] Checking Client
[ESP8266 Driver] Client is inactive.
[ESP8266 Driver] Checking WiFi AP
[ESP8266 Driver] AP configured properly
[ESP8266 Driver] Station IP: 192.168.4.1
[ESP8266 Driver] Checking Server
[ESP8266 Driver] Starting server on port: 7223
[ESP8266 Driver] Opened comm channel 0
GPS: probing baud rate: 115200
GPS: Using baud rate: 115200
GPS: module detected at: 115200
GPS: Configuring baud rate to: 921600
GPS: Message was successfully received.
GPS: Configuring was successful.
GPS: detected current update rate: 10
GPS: Configuring NMEA messages
GPS: Message was successfully received.
GPS: Successfully configured NMEA
GPS: configure message type: win
GPS: configure navigation data message interval: win
GPS: Configuring Gnss Navigation Mode: win
GPS: Disabling NMEA messages
GPS: Message was successfully received.
GPS: Successfully disabled NMEA messages
GPS: provisioned
```
and here is what I see on MK2:

```
GPS: provisioning attempt
GPS: probing baud rate: 921600
[at] Command timed out
[esp8266] Version info:
        AT version:0.23.0.0(Apr 24 2015 21:11:01)
        SDK version:1.0.1
        Ai-Thinker Technology Co. Ltd.
        Apr 27 2015 13:55:14
[esp8266] Initialized
[ESP8266 Driver] Device state: 2
[ESP8266 Driver] Checking WiFi Device...
[ESP8266 Driver] Checking WiFi Device...
[ESP8266 Driver] Checking Client
[ESP8266 Driver] Retrieving Wifi Client Info
[ESP8266 Driver] Checking Client
[ESP8266 Driver] Client is inactive.
[ESP8266 Driver] Checking WiFi AP
[ESP8266 Driver] Retrieving Wifi AP Info
[ESP8266 Driver] AP info:
         SSID      : RaceCapture/Pro MK2 431
         Channel   : 11
         Encryption: 0
[ESP8266 Driver] Checking WiFi AP
[ESP8266 Driver] Retrieving Wifi IP info
[ESP8266 Driver] Checking Client
[ESP8266 Driver] Client is inactive.
[ESP8266 Driver] Checking WiFi AP
[ESP8266 Driver] AP configured properly
[ESP8266 Driver] Station IP: 192.168.4.1
[ESP8266 Driver] Checking Server
[ESP8266 Driver] Starting server on port: 7223
[ESP8266 Driver] Opened comm channel 0
BT: Device responds at baud 115200
BT: Init complete
GPS: probing baud rate: 115200
GPS: probing baud rate: 9600
GPS: Using baud rate: 9600
GPS: module detected at: 9600
GPS: Configuring baud rate to: 921600
GPS: Message was successfully received.
GPS: Configuring was successful.
GPS: detected current update rate: 1
GPS: Configuring update rate to 10: win
GPS: Configuring NMEA messages
GPS: Message was successfully received.
GPS: Successfully configured NMEA
GPS: configure message type: win
GPS: configure navigation data message interval: win
GPS: Configuring Gnss Navigation Mode: win
GPS: Disabling NMEA messages
GPS: Message was successfully received.
GPS: Successfully disabled NMEA messages
GPS: provisioned
```
As you can see RCT came up with a baud of 115200 and MK2 came up with a baud of 9600